hera_cal/vis_clean.py

```diff
@@ -692,14 +915,39 @@ def fourier_filter(self, filter_centers, filter_half_widths, mode,
                                 skipped[:, i] = True
                             elif dim == 1:
                                 skipped[i] = True
-
+            # A redundant system (in the airline sense of the word).
```

Comment:
not sure what you mean here

Reply:
The comment isn't super helpful so I've removed it. Mean that the RMS flagging deals with artifacts that might make it through the other flags (just like airlines have many redundant copies of systems as backups in case one (or more) fail).